### PR TITLE
Moving WebSocket tests to more correct location.

### LINKED ISSUE #425: Need to move WebSocket tests to new correct location.
Need to move the WebSocket tests located...
C:\OSS\wcf\src\System.Private.ServiceModel\tests\Scenarios\Binding\NetHttp\

To...
C:\OSS\wcf\src\System.Private.ServiceModel\tests\Scenarios\Extensibility\WebSockets\

### PR DESCRIPTION
**Pull request by StephenBonikowsky**
* Fixes #425
* Re-named the tests, a couple were inaccurate and one was completely wrong.
* Improved comments.

### CODE REVIEW

**Comment by iamjasonp** [comment]
LGTM - thanks for cleaning this up

File: src/System.Private.ServiceModel/System.Private.ServiceModel.sln [hunk 1]
```diff
@@ -120,10 +120,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Unit", "Unit", "{19B3A13A-9
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "UnitTests.Common", "tests\Common\Unit\UnitTests.Common.csproj", "{E896294A-AB4A-4AF5-A01C-A19E3972EFF9}"
 EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "NetHttp", "NetHttp", "{1A73BF1C-F8DB-47EC-8198-2A6DFFC93333}"
```
Comment (roncain) [inline src/System.Private.ServiceModel/System.Private.ServiceModel.sln]: I see the removed projects.  Should there also be some added projects for them too?

**Comment by roncain** [comment]
Just a reminder -- when these move the TFS ToF wrapper projects may also require renaming or moving.

**Comment by roncain** [comment]
@dotnet-bot please retest

**Comment by StephenBonikowsky** [comment]
@dotnet-bot retest this please

**Pull request merged**